### DISCUSSION TITLE
四月更新 Pro Max

### DISCUSSION THREAD

**Discussion by Richasy** [Richasy/Bili.Uwp#1068] [通知]
说好不更新，又打脸了，谁让滚动卡的我自己都看不下去了呢

顺带一提，如果你喜欢这个应用，可以麻烦在商店留个好评吗？虽然不能公开，但如果在辛苦修完bug后能看到更多的正面反馈，至少能让我觉得我做的事情是有价值的❤️

谢谢各位大佬(♡ ὅ ◡ ὅ )ʃ♡

## 🚀 功能

- 添加评论入口及开发者其他应用的链接 @Richasy (#1066)
- 播放时自动滚动到正在播放的分P/分集 @Richasy (#1064)
- 部分支持显示及播放合集内容 @Richasy (#1060)
    - 有些视频手机上显示有合集（比如木鱼水心的水浒传系列），但通过api拿不到合集信息，具体原因以后再研究

## 🐛 Bug 修复

- 修复在播放结束后有时会卡死的问题 @Richasy (#1067)
- 修复字幕可能会吞字符的问题 @Richasy (#1065)
- 修复播放界面标题栏图标异常 @Richasy (#1063)
- 修复表情在换行边缘会遮挡其他文本的问题 @Richasy (#1061)
- 修复无音轨视频加载失败的问题 @Richasy (#1058)
- 修复命令行播放视频失败的问题 @Richasy (#1057)
- 移除媒体传输控件设置 @Richasy (#1055)
- 滚动性能优化 @Richasy (#1054)

<hr /><em>This discussion was created from the release <a href='https://github.com/Richasy/Bili.Uwp/releases/tag/v2.2204.5.0'>四月更新 Pro Max</a>.</em>

**Comment by msycn** [comment]
请问R大是哪个商店,这里给不了好评,只能点个赞哈哈👍

**Reply by Richasy** [reply]
新版设置页面底部有

**Reply by zijiren233** [reply]
深夜更新，大佬辛苦啦

**Reply by msycn** [reply]
终于恢复上架，以后更新方便多了

**Reply by HaoTian22** [reply]
这是不是说以后能直接在商店更新了？github安装的是要卸载才能切换到商店版吗？

**Reply by Richasy** [reply]
@HaoTian22 看仓库的自述文件

**Reply by ImmortalGlory** [reply]
弱弱地问一句，win10的咋办？

**Comment by silvikid** [comment]
诶嘿,在商店下载的可以看up猪发的文字动态,感谢大佬,已5星

**Reply by Richasy** [reply]
23333，那你可能用的是Win10，下的是旧版本，不过新版会在5月也开始支持更多的动态类型

**Reply by elvisw** [reply]
Win10商店用不了新版么

**Reply by silvikid** [reply]
> 23333，那你可能用的是Win10，下一个旧版本，不过新版会在5月也开始支持更多的动态类型

期待更新

**Reply by silvikid** [reply]
> Win10商店用不了新版么

不知道,没有搜索到

**Comment by DDDandWWW** [comment]
哈哈哈，太棒了，越来越好用了。

**Comment by happylittle2010** [comment]
感谢大佬！！！！感谢！！！（鞠躬）

**Comment by Htskywalker** [comment]
感谢大佬，不过设置里找不到漫游设置

**Reply by Richasy** [reply]
五月再更，四月应该不更新了
